lib/packagekit-glib2/pk-enum.h

```diff
@@ -594,6 +593,7 @@ typedef enum {
 	PK_GROUP_ENUM_ELECTRONICS,
 	PK_GROUP_ENUM_COLLECTIONS,
 	PK_GROUP_ENUM_VENDOR,
+	PK_GROUP_ENUM_DESKTOP_DDE,
```

Comment:
https://github.com/PackageKit/PackageKit/pull/603#discussion_r1193163823 @ximion I put it above `PK_GROUP_ENUM_NEWEST` so `PK_GROUP_ENUM_NEWEST` still means it's the newest. I'm not sure if I understand it correctly, so if it's wrong (if we still need to put it below `PK_GROUP_ENUM_NEWEST` and above `PK_GROUP_ENUM_LAST`), please let me know so I can amend the patch.

btw, I'm at GMT+8 and I have to be away on official business tomorrow, so I will likely not be able to response timely in the next 15 hours. So please forgive me if there are anything need to be done from my side but I cannot response timely. Thanks!

Reply:
Oh, sorry I did misunderstand the purpose of `PK_GROUP_ENUM_NEWEST`. Since it's stand for "Special group for recently updated packages", so we indeed need to move it below `PK_GROUP_ENUM_NEWEST`. Patch updated.

Reply:
Indeed ;-)